code/game/objects/items.dm

```diff
@@ -263,6 +263,14 @@ cases. Override_icon_state should be a list.*/
 // Due to storage type consolidation this should get used more now.
 // I have cleaned it up a little, but it could probably use more.  -Sayu
 /obj/item/attackby(obj/item/W, mob/user)
+	if(istype(W,/obj/item/card/id))
+		if(((flags_item & MOB_LOCK_ON_EQUIP)) && (user == locked_to_mob))
+						if (alert(user, "Would you like to disable the ID lock for \the [src]? This cannot be undone!", "Confirm", "Yes", "No") == "Yes")
+				to_chat(user, SPAN_WARNING("You unlock \the [src]."))
+				src.flags_item &= ~MOB_LOCK_ON_EQUIP
+				locked_to_mob = null
+				log_admin("[user] has removed the ID lock on \the [src]. Owner of the item was [locked_to_mob] (CKEY: ([user.ckey]))")
+			return
```

Comment:
indentation all wacky here, you might want to use defines instead of strings for the Yes and No 🤷

Reply:
weird how it messed up like that
fixed